models/accountModel.js

```diff
@@ -38,7 +38,7 @@ module.exports = function (sequelize, DataTypes) {
             }
         },
         zip: {
-            type: DataTypes.INTEGER,
+            type: DataTypes.STRING,
```

Comment:
Let's keep the zip code as a string; not sure if making it be an integer will cause the zeros to drop as the first character in a five digit zip code (ex.08034)